commands/members/add.js

```diff
@@ -5,16 +5,25 @@ let co     = require('co');
 let extend = require('util')._extend;
 
 function* run (context, heroku) {
+  const FREE_TEAM_LIMIT = 5;
   let org   = context.org;
   let email = context.args.email;
   let role  = context.flags.role;
+  let members = yield heroku.get(`/organizations/${org}/members`);
+  let features = yield heroku.get('/account/features');
+  let orgCreationFeature = features.find(function(feature) {
+    return feature.name === 'standard-org-creation';
+  });
 
   let request = heroku.request({
     method: 'PUT',
     path:   `/organizations/${org}/members`,
     body:   {email, role},
   });
   yield cli.action(`Adding ${cli.color.cyan(email)} to ${cli.color.magenta(org)} as ${cli.color.green(role)}`, request);
+  if ((members.length === FREE_TEAM_LIMIT) && orgCreationFeature) {
```

Comment:
This only should show up if `request` has been successfully. 

If that's not the case, does this code would appear?